### PR TITLE
fix a couple of bits of invalid html

### PR DESCRIPTION
**Pull request by chris48s**
another "leave things a bit better than we found them" PR

This is far from all the invalid HTML, but these 2 were at quick and easy to fix

### CODE REVIEW

File: wcivf/apps/elections/templates/elections/includes/_single_ballot.html [hunk 1]
```diff
@@ -56,7 +56,7 @@ <h2>
                                 </span>
                             </h2>
                         </summary>
-                        <p>{% blocktrans with description=postelection.election.description|markdown %} {{ description }}{% endblocktrans%}</p>
+                        <div>{% blocktrans with description=postelection.election.description|markdown %} {{ description }}{% endblocktrans%}</div>
```
Comment (chris48s) [inline wcivf/apps/elections/templates/elections/includes/_single_ballot.html:59]: nested <p> is invalid and `|markdown` will generate HTML containing `<p></p>`s

Comment (symroe) [reply wcivf/apps/elections/templates/elections/includes/_single_ballot.html:59]: Sorry, what do you mean by "nested p is invalid"? This is a `p` in a `details`, what's wrong with that?

If the `|markdown` is going to make a `p`, can we just remove the wrapping `div`?

Comment (chris48s) [reply wcivf/apps/elections/templates/elections/includes/_single_ballot.html]: There's nothing wrong with a `<p>` in a `<details>`

`<p>` inside another `<p>` is invalid.

`|markdown` will generate at least one `<p>`. I've just removed the container element (`<div>`) completely.

Comment (symroe) [reply wcivf/apps/elections/templates/elections/includes/_single_ballot.html]: Got it, sorry I got confused by that :+1:

**Review by symroe — APPROVED** [review]

**Pull request merged**